<launch>

  <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen" clear_params="true">
    <rosparam file="$(find robotx_nav)/config/costmap_common_params.yaml" command="load" ns="global_costmap" />
    <rosparam file="$(find robotx_nav)/config/costmap_common_params.yaml" command="load" ns="local_costmap" />
    <rosparam file="$(find robotx_nav)/config/local_costmap_params.yaml" command="load" />
    <rosparam file="$(find robotx_nav)/config/global_costmap_params.yaml" command="load" />
    <rosparam file="$(find robotx_nav)/config/base_local_planner_params.yaml" command="load" />
    <remap from="odom" to="odometry/filtered" />
  </node>
  
</launch>
